### ISSUE TITLE
Some error happened during building image at wps mode 

### ISSUE THREAD

**Issue by chibataiki** [ambiot/amb1_sdk#3]
Build Env:
ubuntu 22.04 x86_64 
toolchain:  arm-none-eabi-gcc/4_8-2014q3/
chip: rtl8710af

1.  the header file name is error, correct one is `rtl8195a_otg.h`
```
In file included from ../../../component/common/drivers/usb_class/device/class/msc/inc/usbd_msc.h:5:0,
                 from ../../../component/common/example/usb_mass_storage/example_usb_msc.c:9:
../../../component/soc/realtek/8195a/fwlib/ram_lib/usb_otg/include/usb_gadget.h:533:30: fatal error: rtl8195a_otg_zero.h: No such file or directory
 #include"rtl8195a_otg_zero.h"
                              ^
```

2.  If set WPS on, as /component/common/example/wlan_scenario/readme.txt ，
```
../../../component/common/api/wifi/wifi_conf.c:3247:1: error: invalid initializer
 char wps_profile_ssid[33]=0;
 ^
../../../component/common/api/wifi/wifi_conf.c:3248:1: error: invalid initializer
 char wps_profile_password[65]=0;
make[1]: *** [application.mk:782: ../../../component/common/api/wifi/wifi_conf.o] Error 1

```
3. DB_RAM overflow ,   as problem 2, if set WPS on, as /component/common/example/wlan_scenario/readme.txt 
```
../arm-none-eabi/bin/ld: application/Debug/bin/application.axf section `.bf_data' will not fit in region `BD_RAM'
../arm-none-eabi/bin/ld: region RAM overflowed with stack
../arm-none-eabi/bin/ld: region `BD_RAM' overflowed by 13084 bytes
```

**Comment by Aurical** [comment]
1. This is caused by an error in the filename `Rtl8195a_otg_zero`. The uppercase `R` works in Windows but will not work in Linux. Renaming the file to use a lowercase `r` will solve the error. This will be fixed in the next commit.
2. This is an actual mistake. Adding `{``}` around `0` will solve the error. This will be fixed in the next commit.
3. This is caused by hardware limitations. Enabling WPS and other features will use up more RAM, which in this case exceeds the available RAM. You can try to solve this by editing the linker file `rlx8195A-symbol-v02-img2.ld` and modifying what needs to go into RAM, or reduce the heap size in `FreeRTOSConfig.h` by changing `#define configTOTAL_HEAP_SIZE			( ( size_t ) ( 45 * 1024 ) )`

**Comment by M-ichae-l** [comment]
For 1 and 2 has been resolved by commit 808458eec488c90ec6c237e380f8b52cff9e9550 and 63d54848534113c50e4e78d0ba45359be9ef7e0f
